### ISSUE TITLE
NPE on new player connect

### ISSUE THREAD

**Issue by delbertina** [LuckPerms/ExtraContexts#39]
Seems to be the same issue as #34 

I saw the error happen when a new player joined the server. A regular had already joined before this player did w/o the spam. So the issue could just be intermittent or it could just be an issue with new players.

[ Log](https://pastebin.com/AJjnDP8S)
 (I had to remove some of the repeated msgs towards the end to be under pastbin's limit)
<details>
  <summary>Config</summary>
  
 ```
 # ExtraContexts
# made with <3 by Luck


# Set a provider to true to enable it.
# Contexts are explained in more detail here:
# https://github.com/lucko/LuckPerms/wiki/Context


# Provides the 'worldguard:region' and 'worldguard:in-region' contexts.
# Returns the name of each WorldGuard region the player is currently in.
#
# e.g. worldguard:in-region=true and worldguard:region=spawn
worldguard-region: false

# Provides the 'worldguard:flag-xxx' contexts.
# Returns the value of each flag set by WorldGuard in the region the player is currently in.
#
# e.g. worldguard:flag-build=allow
worldguard-flag: true

# Provides the 'whitelisted' context.
# Returns if the player is whitelisted on the server or not.
#
# e.g. whitelisted=true
whitelisted: false

# Provides the 'team' context.
# Returns the name of the team the players is in.
#
# e.g. team=pvp-blue
team: false

# Provides the 'has-played-before' context.
# Returns if the player has connected to the server before, or if this is their first time.
#
# e.g. has-played-before=false
has-played-before: false

placeholderapi: false
placeholderapi-placeholders:
  allowflight: "%player_allow_flight%"
```
</details>

Luckperms 5.3.86
ExtraContexts Build # 17 from Jenkins (newest)
WorldGuard 7.0.6
Paper 1.18.1 # 134

**Comment by Sxtanna** [comment]
This is caused by ExtraContexts using an outdated version of WorldGuardWrapper that returns null when encountering an "unsupported" region flag.

This was fixed in WGW a very long time ago, idk why EC hasn't updated to use it yet.

Offending code:
https://github.com/CodeMC/WorldGuardWrapper/blob/15301c540f5fb65f399541050b6730e699bf9200/implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java#L106-L111

Fixed code:
https://github.com/CodeMC/WorldGuardWrapper/blob/4ea70913c56765bf2d0bec2035d6483e27f57098/implementation/v7/src/main/java/org/codemc/worldguardwrapper/implementation/v7/WorldGuardImplementation.java#L106-L111

**Comment by Sxtanna** [comment]
This is fixed in PRs #40 and #41 using alternate solutions.

**Comment by lucko** [comment]
Thanks @Sxtanna
